cdap-docs/cdap-apps/source/tracker/index.rst

```diff
@@ -0,0 +1,441 @@
+  to help locate datasets.
+
+- Through its use of lineage, it lets you understand the impact of changing a dataset on
+  other datasets, processes or queries.
+
+- Tracks the flow of data across enterprise systems and data lakes.
+
+- Provides trusted and complete metadata on datasets, enabling traceability to resolve
+  data issues and to improve data quality.
+
+
+**Supports Standardization, Governance, and Compliance Needs**
+
+- Provide IT with the traceability needed in governing datasets and in applying compliance
+  rules through seamless integration with other extensions.
+
+- Tracker has consistent definitions of metadata-containing information about the data to reconcile
+  differences in terminologies.
+
+- It helps you in the understanding of the lineage of your business-critical data.
+
+
+**Blends Metadata Analytics and Integrations**
+
+- See how your datasets are being created, accessed, and processed with built-in usage
+  analytics capabilities.
+
+- Multi-dimensional usage analytics to understand the interaction between users,
+  applications, and datasets.
+
+- Extensible integrations are available with enterprise-grade MDM (master data management)
+  systems such as `Cloudera Navigator <https://www.cloudera.com/products/cloudera-navigator.html>`__ 
+  for centralizing metadata repository and the delivery of complete, accurate, and correct
+  data.
+
+
+Example Use Case
+----------------
+An example use case describes how Tracker was employed in the `data cleansing and validating of
+three billion records <http://customers.cask.co/rs/882-OYR-915/images/tracker-casestudy1.pdf>`__.
+
+
+Tracker Application
+===================
+The Cask Tracker application consists of an application in CDAP with two programs and two datasets:
+
+- ``_Tracker`` application: names begins with an underscore
+- ``AuditLog`` service: it exposes the Tracker audit log as an API
+- ``AuditLogFlow`` flow: subscribes to Kafka audit messages and stores them in the ``AuditLog``	dataset
+- ``AuditLog`` custom dataset: type ``co.cask.tracker.entity.AuditLogTable``
+- ``kafkaOffset`` dataset: type key-value table
+
+The Tracker UI is shipped with CDAP, started automatically as part of the CDAP UI, and is available at:
+
+  http://localhost:9999/ns/default/tracker/home
+  
+or (Distributed CDAP):
+
+  http://host:dashboard-bind-port/ns/default/tracker/home
+  
+
+Tracker is built from a system artifact included with CDAP, |literal-cask-tracker-version-jar|.
+
+
+Installation
+============
+Cask Tracker is deployed from its system artifact included with CDAP. A CDAP administrator
+does not need to build anything to add Cask Tracker to CDAP; they merely need to add two
+properties to the ``cdap-site.xml`` and the enable the the application after starting CDAP.
+
+These two properties are required in the ``cdap-site.xml`` for the audit log functionality of Tracker::
+  
+  <!-- Audit Configuration -->
+
+  <property>
+    <name>audit.enabled</name>
```

Comment:
I also think they just changed it to be enabled by default: https://github.com/caskdata/cdap/pull/5650